<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HUYF9-PF1L46YU" ignored="0" name="Test" tests="2" failures="2" timestamp="16 一月 2020 02:43:44 GMT" time="0.789" errors="0">
  <testcase name="test1" time="0.349" classname="timeout.TestTimeOut">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method timeout.TestTimeOut.test1() didn&amp;apos;t finish within the time-out 240">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method timeout.TestTimeOut.test1() didn't finish within the time-out 240
at java.lang.Throwable.printStackTrace(Throwable.java:643)
at java.lang.Throwable.printStackTrace(Throwable.java:634)
at timeout.TestTimeOut.test1(TestTimeOut.java:13)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="0.242" classname="timeout.TestTimeOut">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method timeout.TestTimeOut.test2() didn&amp;apos;t finish within the time-out 240">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method timeout.TestTimeOut.test2() didn't finish within the time-out 240
at java.lang.Thread.sleep(Native Method)
at timeout.TestTimeOut.test2(TestTimeOut.java:21)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Test -->
